jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1905,6 +1905,20 @@ java_bytecode_parsert::parse_method_handle(const method_handle_infot &entry)
   std::replace(class_name.begin(), class_name.end(), '.', '$');
   // replace '/' for '.' (package)
   std::replace(class_name.begin(), class_name.end(), '/', '.');
+  if(
+    entry.get_reference_kind() ==
+      method_handle_infot::method_handle_kindt::REF_getField ||
+    entry.get_reference_kind() ==
+      method_handle_infot::method_handle_kindt::REF_putField ||
+    entry.get_reference_kind() ==
+      method_handle_infot::method_handle_kindt::REF_getStatic ||
+    entry.get_reference_kind() ==
+      method_handle_infot::method_handle_kindt::REF_putStatic)
+  {
+    // Field accessors not implemented yet
```

Comment:
Expanded comment; in brief these are notional "field bindings", e.g. you would be able to say `Producer<Integer> = this::someIntegerField` and get a REF_getField instead of a REF_invokeSpecial pointing at a short function that fetches the field. While it made it into the JVMS and might be used in future, AFAICT nobody has actually done that.

Reply:
lol cool feature I guess :sunglasses: